Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java

```diff
@@ -151,6 +160,22 @@ public void onDateChanged(DateEditText view, Date date) {
 				startDateField.setMaxDate(date);
 			}
 		});
+		
+		userField = (TextView)headerView.findViewById(R.id.tv_user);
+		userField.append(claim.getUser().getName());
+		
+		approverField = (TextView)headerView.findViewById(R.id.tv_approver);
+		try{
```

Comment:
May be cleaner to check if `claim.getApprover()` returns `null` and proceeding accordingly than to catch the exception in that case.